installation/on_opensuse.html

```diff
@@ -0,0 +1,32 @@
+# On OpenSUSE
+
+In OpenSUSE, you can use the official Crystal repository.
```

Comment:
I'd prefer a more descriptive writing style instead of directly addressing the reader in every sentence.
Example:
```
On OpenSUSE, Crystal can be installed from the official rpm package using Zypper.
```